package.json

```diff
@@ -6,10 +6,13 @@
   "type": "module",
   "scripts": {
     "start": "npm run dev",
+    "start:test": "NODE_ENV=testing npm run dev",
     "lint": "eslint '**/*.js'",
     "lint:fix": "eslint '**/*.js' --fix",
     "build": "vite build",
-    "dev": "vite dev --host"
+    "dev": "vite dev --host",
+    "test": "backstop --config=backstop.cjs --docker test",
+    "postinstall": "npx playwright install chromium"
```

Comment:
Maybe we should make this somewhat optional? Now it will be pulling in several mbs on each install for playwright while it may not always be used

Reply:
You are right, we should not install a browser by default. I think a separate test script can do that as you suggested in the other comment.